scripts/data_guide/data_guide.qmd

```diff
@@ -0,0 +1,620 @@
+      caption = 'Sequence Assay Genomic Information',
+     booktabs = T, longtable = T,
+      row.names = F, col.names = c("Sequencing Assay", "Calling Strategy", "Number of genes", "Target Capture Kit")) %>% kable_styling(latex_options = c("repeat_header", "striped"), font_size=5)
+```
+
+# Genomic Profiling at Each Center
+
+```{r}
+#| eval: true
+#| echo: false
+#| results: asis
+
+for (center in sort(processed_centers)) {
+  center = tolower(center)
+  a <- knitr::knit_child(glue::glue('genomic_profiles/{center}.qmd'), quiet=T)
+  cat(a, sep="\n")
+}
+
+```
+
+# Description of Data Files
+
+Description on most of the data files can be found under the "Data files" section for [cBioPortal file formats](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats).
+
+
+data_mutations_extended.txt
+
+* Description: Mutation data
+* Details: [MAF format](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#mutation-data)
+
+data_CNA.txt
+
+* Description: Discritized copy number data
+* Details: [CNA format](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#discrete-copy-number-data)
+
+data_sv.txt
+
+* Description: Structural variant data
+* Details: [Structural Variant format](https://docs.cbioportal.org/file-formats/#structural-variant-data)
+
+genomic_information.txt
+
+* Description: File describing genomic coordinates covered by all platforms for GENIE data. Note: Not all centers contributed copy number data to GENIE
+* Details: This is not a cBioPortal file format.
+  * Chromosome, Start_Position, End_Position: Gene positions
+  * Hugo_Symbol: Re-mapped gene symbol based on gene positions
+  * ID: Center submitted gene symbols
+  * SEQ_ASSAY_ID: The institutional assay identifier for genomic testing platform. Feature_Type: "exon", "intron", or "intergenic"
+  * includeInPanel: Used to define gene panel files for cBioPortal. clinicalReported: These are the genes that were clinically Reported. Blank means information not provided.
+
+assay_information.txt
+
+* Description: Assay information
+* Details: This is not a cBioPortal file format
+  * is_paired_end, library_selection, library_strategy, platform, read_length, target_capture_kit, instrument_model: defined by [GDC read group](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group)
+  * number_of_genes: Number of genes from which variants are called
+  * variant_classifications: List of types of variants that are reported for this assay.
+  * gene_padding: Number of base pairs to add to exon endpoints for the inBED filter.
+  * alteration_types: List of alteration types. specimen_type: List of specimen types.
+  * specimen_tumor_cellularity: Tumor Cellularity Cutoff
+  * calling_strategy: Tumor only or tumor normal.
+  * coverage: List of coverage
+
+data_cna_hg19.seg
+
+* Description: Segmented copy number data. Note: Not all centers contributed copy number data to GENIE
```

Comment:
A SEG file (segmented data; .seg or .cbs) is a tab-delimited text file that lists loci and associated numeric values.